src/Features/Core/Portable/EmbeddedLanguages/IEmbeddedLanguageFeatures.cs

```diff
@@ -18,15 +18,5 @@ internal interface IEmbeddedLanguageFeatures : IEmbeddedLanguage
         /// A optional highlighter that can highlight spans for an embedded language string.
         /// </summary>
         IDocumentHighlightsService DocumentHighlightsService { get; }
-
-        /// <summary>
-        /// An optional completion provider that can provide completion items for this
-        /// specific embedded language.
-        /// 
-        /// <see cref="AbstractEmbeddedLanguageCompletionProvider"/> will aggregate all these
-        /// individual providers and expose them as one single completion provider to
-        /// the rest of Roslyn.
-        /// </summary>
-        CompletionProvider CompletionProvider { get; }
```

Comment:
so this basically defeats the purpose of the design, where an embedded language can supply this little interface and then plug into all our systems easily.  we should discuss this.

Reply:
discussed offline, switched to different approach